crypto-rng.cabal

```diff
@@ -14,7 +14,7 @@ maintainer:          Mikhail Glushenkov <mikhail.glushenkov@scrive.com>,
 copyright:           Scrive AB
 category:            Crypto
 build-type:          Simple
-tested-with:         GHC ==8.0.2 || ==8.2.2 || ==8.4.4 || ==8.6.5 || ==8.8.3 || ==8.10.1
+tested-with:         GHC ==8.8.4 || ==8.10.7 || ==9.0.2
```

Comment:
No 9.2.1 here?

Reply:
No, one of the dependencies doesn't compile.